src/Components/Endpoints/src/Rendering/EndpointHtmlRenderer.Streaming.cs

```diff
@@ -246,7 +248,7 @@ private void WriteComponentHtml(int componentId, TextWriter output, bool allowBo
             output.Write("-->");
         }
 
-        if (endMarkerOrNull is { } endMarker)
+        if (endMarkerOrNull is { } endMarker && !_isHandlingErrors)
```

Comment:
What is the purpose of disabling these markers when processing errors?

Reply:
To render pure HTML without any additional markup that won't be used.

Reply:
I understand. However since we don't need to aggressively optimize the payload size of the error page, would it be OK not to make this logic deal with varying its output?

I appreciate there's a balance to be struck and think that it would be preferable for other code not to have to have special cases for this unless we're really required to do so. Just trying to limit complexity growth!

Reply:
```suggestion
        if (endMarkerOrNull is { } endMarker)
```

Reply:
Sorry if I was unclear but I also meant removing the other `_isHandlingErrors` checks above in this file, and in fact also removing:

 * The `_isHandlingErrors` field, since it would no longer be used
 * The `isHandlingErrors` parameter, since it would no longer be used